public/app/features/alerting/unified/mocks/server/configure.ts

```diff
@@ -113,6 +115,10 @@ export function mimirDataSource() {
   return { dataSource };
 }
 
+export function setPrometheusRules(ds: DataSourceInstanceSettings, groups: PromRuleGroupDTO[]) {
+  server.use(http.get(`/api/prometheus/${ds.uid}/api/v1/rules`, paginatedHandlerFor(groups)));
+}
```

Comment:
I think a TODO should be to make the `/api/prometheus/:datasourceUid/api/v1/rules` endpoint behave differently within the handler, depending on the UID passed. So that we have a datasource that has pagination setup, and one that doesn't (and all of the branching logic for this just sits inside the one handler?)